### PR TITLE
[WIP] Fix merge

### PR DESCRIPTION
**Pull request by smeenai**
Deal with the GetSymbolVendor removal.

The GetASTData changes are incorrect, since SymbolVendor::GetASTData has
some extra logic that's not in SymbolFile. This gets things to build,
but GetASTData needs to be properly addressed.

### CODE REVIEW

File: source/Target/SwiftLanguageRuntime.cpp [hunk 1]
```diff
@@ -3068,7 +3068,7 @@ void SwiftLanguageRuntime::FindFunctionPointersInCall(
                   ExecutionContext exe_ctx;
                   frame.CalculateExecutionContext(exe_ctx);
                   error = argument_values.GetValueAtIndex(0)->GetValueAsData(
-                      &exe_ctx, data, 0, NULL);
+                      &exe_ctx, data, NULL);
```
Comment (smeenai) [inline source/Target/SwiftLanguageRuntime.cpp:3071]: This one is unrelated to the GetSymbolVendor removal ... the API to GetValueAsData was changed.

**Comment by JDevlieghere** [comment]
Correct resolution in https://github.com/apple/swift-lldb/pull/1868
